apps/google-analytics-4/lambda/Makefile

```diff
@@ -5,6 +5,10 @@ all: help
 start-app:
 	@docker compose up --build
 
+## make go: Starts a docker image for dynamodb, our serverless lambda, and frontend for local development
```

Comment:
should we remove start-app then? What's the difference?

Reply:
i wanted to make it backward compatible, but i'd like to :)

Reply:
💣 💥 Nuke it!

Reply:
removed `start-app`